ci/install-package-dependencies.sh

```diff
@@ -134,3 +137,8 @@ tar -C /tools/verible -xf "$verible_tar" --strip-components=1
 
 # Propagate PATH changes to all subsequent steps of the job
 echo "##vso[task.setvariable variable=PATH]/tools/verible/bin:$PATH"
+
+# Install Rust
+sw/vendor/rustup/rustup-init.sh -y \
+    --default-toolchain "${RUST_VERSION}"
+export PATH="$HOME/.cargo/bin:$PATH"
```

Comment:
To make PATH available to subsequent build jobs in the pipeline you need to use the approach above `task.setvariable`.